delta/app/src/main/scala/ch/epfl/bluebrain/nexus/delta/routes/ResourcesRoutes.scala

```diff
@@ -0,0 +1,196 @@
+import ch.epfl.bluebrain.nexus.delta.sdk.error.ServiceError.AuthorizationFailed
+import ch.epfl.bluebrain.nexus.delta.sdk.model.IdSegment.{IriSegment, StringSegment}
+import ch.epfl.bluebrain.nexus.delta.sdk.model.projects.ProjectRejection.ProjectNotFound
+import ch.epfl.bluebrain.nexus.delta.sdk.model.projects.{ProjectRef, ProjectRejection}
+import ch.epfl.bluebrain.nexus.delta.sdk.model.{BaseUri, IdSegment, Label}
+import ch.epfl.bluebrain.nexus.delta.sdk._
+import ch.epfl.bluebrain.nexus.delta.sdk.model.acls.AclAddress
+import ch.epfl.bluebrain.nexus.delta.sdk.model.identities.Caller
+import ch.epfl.bluebrain.nexus.delta.sdk.model.resources.ResourceRejection
+import io.circe.Json
+import kamon.instrumentation.akka.http.TracingDirectives.operationName
+import monix.execution.Scheduler
+
+/**
+  * The resource routes
+  *
+  * @param identities the identity module
+  * @param acls       the ACLs module
+  * @param projects   the projects module
+  * @param resources  the resources module
+  */
+final class ResourcesRoutes(
+    identities: Identities,
+    acls: Acls,
+    projects: Projects,
+    resources: Resources
+)(implicit baseUri: BaseUri, s: Scheduler, cr: RemoteContextResolution, ordering: JsonKeyOrdering)
+    extends AuthDirectives(identities, acls)
+    with CirceUnmarshalling {
+
+  import baseUri.prefixSegment
+
+  private val resourceSchema = IriSegment(schemas.resources)
+
+  def routes: Route =
+    baseUriPrefix(baseUri.prefix) {
+      extractCaller { implicit caller =>
+        pathPrefix("resources") {
+          concat(
+            // SSE resources
+            (pathPrefix("events") & pathEndOrSingleSlash) {
+              get {
+                operationName(s"$prefixSegment/resources/events") {
+                  authorizeFor(AclAddress.Root, events.read).apply {
+                    lastEventId { offset =>
+                      emit(resources.events(offset))
+                    }
+                  }
+                }
+              }
+            },
+            projectRef { ref =>
+              routesForProject(ref)
+            },
+            (uuid & uuid) { (orgUuid, projectUuid) =>
+              fetchProject(orgUuid, projectUuid) { project =>
+                routesForProject(project.value.ref)
+              }
+            }
+          )
+        }
+      }
+    }
+
+  private def routesForProject(ref: ProjectRef)(implicit caller: Caller): Route =
+    concat(
+      (pathPrefix("events") & pathEndOrSingleSlash) {
+        get {
+          operationName(s"$prefixSegment/resources/{org}/{project}/events") {
+            authorizeFor(AclAddress.Project(ref), events.read).apply {
+              lastEventId { offset =>
+                emit(resources.events(ref, offset).leftWiden[ResourceRejection])
+              }
+            }
+          }
+        }
+      },
+      // Create a resource without schema nor id segment
+      (post & noParameter("rev") & pathEndOrSingleSlash & entity(as[Json])) { source =>
```

Comment:
I think currently we also allow `POST /resources/{org}/{proj}/_`

Reply:
Yes, this is already supported: https://github.com/BlueBrain/nexus/blob/74b8a7ec73bd1866e8c65b6b0a8cbe0529b6b3a6/delta/app/src/main/scala/ch/epfl/bluebrain/nexus/delta/routes/ResourcesRoutes.scala#L106